src/features/page/home/dialogs/CompareSimulationsDialog.tsx

```diff
@@ -48,21 +51,31 @@ export default function CompareSimulationsDialog({
           To compare simulations, select a map and then 2 simulations which you
           want to compare
         </DialogContentText>
-        <SpacedInput marginTop={30} label="Map" value={mapId} setValue={setMapId}>
+        <LabeledInput
+            marginTop={30}
+            label="Map"
+            disabled = { !mapData }
+            value={mapId}
+            setValue={setMapId}
+            error={isError}
+            helperText={!mapData ? 'Couldn\'t load maps. Check your connection.' :
```

Comment:
czemu tu akurat po mapData?

Reply:
Bo to jest stan pierwszego modala i jakby jest fetchowane jako pierwsze, wiec jezeli nie ma tam wartosci odpowiedniej to znaczy ze backend nie zwaca odp (np heroku jeszcze wstaje)

Reply:
dobra przerobiłem faktycznie